app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -1222,7 +1230,21 @@ class WithdrawModalNew extends React.Component {
                                 }
                             </div>
                         ) : null}
-
+                        
+                        {/*TAG*/}
```

Comment:
both tag and memo are triggered by `supportsMemos`, can you explain difference please?

Reply:
I do support memos and tags; some others support memos only, so I did backingAsset.memoType === "tagid" now; it will be triggered only when memo type is tagid.

Reply:
I dont understand. What is tag for?

Reply:
For an XRP Ledger address that may receive payments intended for several customer accounts, receiving a payment without a destination tag can be a problem: it is not immediately obvious which customer to credit, which can require a manual intervention and a discussion with the sender to figure out who was the intended recipient. To reduce cases like this, you can enable the RequireDest setting. That way, if a user forgets to include a destination tag in a payment, the XRP Ledger rejects their payment instead of giving you money you don't know what to do with. The user can then send the payment again, using the tag as they should have.

https://xrpl.org/source-and-destination-tags.html

Reply:
> I dont understand. What is tag for?

Every centralized exchange will give you "XRP Address and Tag" both are mandatory for your deposit at centralized exchange, this is in case user is withdrawing from ioxbank.

Reply:
Unlike BitShares send transaction operation; XRPL send transaction operation has both memo and tag, tag has to be a number and memo can be any text.

All centralized exchange are depending on TAG to redirect the XRPL transaction to their corresponding users but memo is not mandatory being used in centralized exchange wallet.

Reply:
@sschiessl-bcp anything else is missing here?